atef/procedure.py

```diff
@@ -44,6 +44,12 @@ def __init__(self):
         self.RE.subscribe(self.db.insert)
 
 
+def walk_steps(step: ProcedureStep) -> Generator[ProcedureStep, None, None]:
```

Comment:
A few functions in this PR could use docstrings. I know they're simple methods in general, but let's just take care of it now before we fall behind.

Reply:
100%, I neglected to do my typical run-through after the methods/functions mostly crystalized.  [ae6bf9e](https://github.com/pcdshub/atef/pull/155/commits/ae6bf9e7a5f55587a057ea21b338215c1a97496a)